webapp/channels/src/components/common/hooks/useOpenSalesLink.ts

```diff
@@ -31,8 +32,9 @@ export default function useOpenSalesLink(): [() => void, string] {
     }
 
     const contactSalesLink = buildMMURL(LicenseLinks.CONTACT_SALES, firstName, lastName, companyName, customerEmail, utmSource, utmMedium);
-    const goToSalesLinkFunc = () => {
+    const goToSalesLinkFunc = useCallback(() => {
         goToMattermostContactSalesForm(firstName, lastName, companyName, customerEmail, utmSource, utmMedium);
-    };
+    }, [companyName, customerEmail, firstName, lastName, utmMedium]);
```

Comment:
Could you revert these changes and submit them in a separate PR? Also, this one is missing the `utmSource`

Reply:
https://github.com/mattermost/mattermost/pull/28868

Re: utmSource
It is a string constant, and apparently the linter is smart enough to consider it "always the same". Nevertheless, I moved it out of the component. That being said... I have a gut feeling utmSource is going to be the same always, so we probably shouldn't need to pass it, and should make the constant down the line.